ts-packages/web/src/app/spaces/deliberation/[id]/page.tsx

```
@@ -0,0 +1,5 @@
+import DeliberationSpacePage from './components/main';
+
+export default async function Page({}: { params: Promise<{ id: string }> }) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)